server/etcdserver/api/v3rpc/watch.go

```diff
@@ -346,8 +346,9 @@ func (sws *serverWatchStream) recvLoop() error {
 			}
 		default:
 			// we probably should not shutdown the entire stream when
-			// receive an valid command.
+			// receive an invalid command.
 			// so just do nothing instead.
+			sws.lg.Warn("invalid watch request received in gRPC stream")
```

Comment:
@spacewander thanks! Logging it for information purpose sounds good, and so I think the severity should be `Info` (not warning).